### PR TITLE
verdict eve field - 6.0.x backports - v2

### PR DESCRIPTION
**Pull request by jufajardini**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket: https://redmine.openinfosecfoundation.org/issues/5794

Describe changes:
- remove section about modbus eve output
- rebase

```
SV_BRANCH=pr/1336
```

https://github.com/OISF/suricata-verify/pull/1336

### CODE REVIEW

File: doc/userguide/output/eve/eve-json-format.rst [hunk 1]
```diff
@@ -1805,3 +1849,184 @@ Example of HTTP2 logging, of a request and response:
       ]
     }
   }
+
+
+Event type: IKE
+---------------
+
```
Comment (jufajardini) [inline doc/userguide/output/eve/eve-json-format.rst:1856]: I checked SV tests and the outputs, and IKE and DHCP appear there. Didn't check for each independent field, though...

Comment (victorjulien) [reply doc/userguide/output/eve/eve-json-format.rst:1856]: someone will need to check :)

Comment (jufajardini) [reply doc/userguide/output/eve/eve-json-format.rst:1856]: Roger that 🙇🏽

**Comment by jufajardini** [comment]
Replaced by: https://github.com/OISF/suricata/pull/9311

**Pull request closed without merging**